### DISCUSSION TITLE
Tray Icon, change font, color, size ?

### DISCUSSION THREAD

**Discussion by SMN204** [Rem0o/FanControl.Releases#2252] [General]
Is it possible to change the size and font of the Temperature tray icons?
![FanControl Tray](https://github.com/Rem0o/FanControl.Releases/assets/60313040/efc213ad-fef1-4b66-86f7-9fcd4af284fb)

**Comment by Rem0o** [comment]
Not as of now.

**Comment by akbyrd** [comment]
+1 for size and font control. It would be great to be able to make them a little smaller and sleeker.
